.github/generate-job-matrix.py

```diff
@@ -18,25 +98,31 @@ def make_gcc_config(version: int) -> Configuration:
             cc=f"gcc-{version}",
             cxx=f"g++-{version}",
         ),
-        cxx_modules=False,
-        std_format_support=version >= 13,
+        feature_support=Features(
+            std_format=version >= 13,
+            freestanding=True,
+        ),
     )
 
 
 def make_clang_config(
-    version: int, platform: typing.Literal["x86-64", "arm64"] = "x86-64"
-) -> Configuration:
+    version: int, architecture: typing.Literal["x86-64", "arm64"] = "x86-64"
+) -> Platform:
     cfg = SimpleNamespace(
-        name=f"Clang-{version} ({platform})",
+        name=f"Clang-{version} ({architecture})",
         compiler=SimpleNamespace(
             type="CLANG",
             version=version,
         ),
         lib="libc++",
-        cxx_modules=version >= 17,
-        std_format_support=version >= 17,
+        feature_support=Features(
+            cxx_modules=version >= 17,
+            std_format=version >= 17,
+            import_std=version >= 17,
```

Comment:
According to our [compatiblity table](https://mpusz.github.io/mp-units/latest/getting_started/cpp_compiler_support/):

```suggestion
            import_std=version >= 18,
```